extras/analyze-ci-crash.sh

```
@@ -0,0 +1,222 @@
+		echo "Warning: Could not find binary '$exe_name' in build artifacts"
+		echo "Attempting analysis without symbols..."
+		BINARY=""
+	else
+		echo "Binary: $BINARY"
+	fi
+
+	# Run gdb in the container for accurate library resolution
+	echo ""
+	echo "--- Backtrace ---"
+
+	GDB_COMMANDS=$(
```

Shellcheck (0.11.0)

[warning] GDB_COMMANDS appears unused. Verify use (or export if used externally).

(SC2034)